src/components/StabilityFeeAlert.jsx

```diff
@@ -9,22 +9,15 @@ import InlineNotification from "./InlineNotification";
 @inject("network")
 @observer
 class StabilityFeeAlert extends React.Component {
-  constructor(props){
-    super(props);
-    this.state = { show: true }
-  }
-
   render() {
     return (
       this.props.content.showStabilityFeeAlert() &&
-        this.state.show &&
-        !localStorage.getItem(`StabilityFeeChangeAlertClosed-${this.props.content.stabilityFeeMarkdown()}`) &&
         this.props.network.isConnected &&
         this.props.network.defaultAccount &&
```

Comment:
still not sure why these two conditions are here :p

Reply:
https://github.com/makerdao/scd-cdp-portal/blob/2e3bdce90066346cf0dba563d71ef4424c0d4522/src/components/Home.jsx#L64

Should be the inverse of that to only render on the dashboard page as opposed to the landing